xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+        and <literal>container image registry</literal> for container images
+        as well as a webserver hosting the <literal>Helm chart repository</literal>
+        files.
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          <literal>1</literal> Host machines for the mirror servers.
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            500 GB Storage
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </itemizedlist>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <important>
+     <title>Adjust Number Of Mirror Servers</title>
+     <para>
+      This scenario description does not contain any fallback contingencies for
+      the mirror servers. Add additional mirror servers (behind a loadbalancer)
+      if you require additional reliability/availability.
+     </para>
+    </important>
+    <procedure>
+     <title>Provision Mirror Servers</title>
+     <step>
+      <para>
+       <link xlink:href="https://www.suse.com/documentation/sles-15/book_quickstarts/data/art_sle_installquick.html">Set
+       up two &sls; 15 machines</link> one on the internal network and one on
+       the airgapped network.
+      </para>
+     </step>
+     <step>
+      <para>
+       Make sure you have
+       <link xlink:href="https://www.suse.com/documentation/sles-15/book_sles_docker/data/preparation.html">enabled
+       the <filename>Containers</filename> module</link> on each server.
+      </para>
+     </step>
+     <step>
+      <para>
+       You will need SSL/TLS certificates to secure services on each server.
+       Make sure the certificates are available system wide so they can be used
+       by the deployed components.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.network">
+    <title>Networking</title>
+    <para>
+     All members of the &productname; cluster must be able to communicate with
+     the internal mirror server(s) within the airgapped network.
+    </para>
+    <para>
+     Ports:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       80 HTTP - &rmtool; Server and Helm chart repository mirror
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       5000 HTTPS - Container image registry
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     The external mirror server must be able to communicate with upstream
+     sources on the internet. The minimal required ports are <literal>80</literal>
+     and <literal>443</literal>.
```

Comment:
This one in particular reads as if port 80 and 443 needs to be opened on the external mirror whereas it truly means that the outgoing traffic on 80 and 443 must be allowed. Maybe we can clarify that.